proto/permission/exported/v1beta1/types.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+package permission.exported.v1beta1;
+
+option go_package = "github.com/axelarnetwork/axelar-core/x/permission/exported";
+
+import "gogoproto/gogo.proto";
+
+option (gogoproto.goproto_getters_all) = false;
+
+enum Role {
+  option (gogoproto.goproto_enum_prefix) = false;
+  option (gogoproto.goproto_enum_stringer) = true;
+
+  ROLE_UNSPECIFIED = 0;
+  ROLE_GOVERNANCE = 1 [ (gogoproto.enumvalue_customname) = "Governance" ];
+  ROLE_CONTROLLER = 2 [ (gogoproto.enumvalue_customname) = "Controller" ];
```

Comment:
don't give them custom names if they live in the `types` package, because then it would be called like `types.Controller` which makes it more difficult to read the code